core/domain/entities/routing/handlers/shared/RegularRequests.php

```diff
@@ -120,7 +120,7 @@ protected function registerDependencies()
             'EventEspresso\core\domain\entities\routing\handlers\admin\EspressoEventsAdmin'  => $admin,
             'EventEspresso\core\domain\entities\routing\handlers\admin\EspressoEventEditor'  => $admin,
             'EventEspresso\core\domain\entities\routing\handlers\admin\EspressoLegacyAdmin'  => $admin,
-            'EventEspresso\core\domain\entities\routing\handlers\admin\GutenbergEditor'  => $admin,
+            'EventEspresso\core\domain\entities\routing\handlers\admin\GutenbergEditor'      => $admin,
```

Comment:
I don't know why this is done in PHP, but it looks cool to have the right side perfectly aligned :+1:

Reply:
just done for readability

Reply:
PHPCS === ESLint